README.md

```diff
@@ -117,6 +117,29 @@ To install these plugins, just copy the two executables (`protoc-gen-swift` and
 that is part of your `PATH` environment variable. Alternatively the full path to
 the plugins can be specified when using `protoc`.
 
+Alternatively, you can get the latest precompiled version of the plugins by adding
+the following line to your `Podfile`:
+
+```ruby
+    pod 'gRPC-Swift-Plugins'
+```
+
+and running
+
+```
+  protoc <Protobuf Definition File> \
+		--plugin=Pods/gRPC-Swift-Plugins/protoc-gen-swift \
+		--swift_out=ProtobufGeneratedCode/
+```
+
+and
+
+```
+protoc <Protobuf Definition File> \
+ --plugin=Pods/gRPC-Swift-Plugins/protoc-gen-grpc-swift \
+ --grpc-swift_out=GRPCGeneratedCode/
+```
```

Comment:
I don't think we need examples of running `protoc` here; saying that the precompiled binaries are available via that pod is fine.